.github/workflows/test.yaml

```diff
@@ -138,7 +138,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Check for Dockerfiles
         id: dockerfiles
-        run: echo -n "found=$(find . -name "*Dockerfile" -printf "%p ")" >> $GITHUB_OUTPUT
+        run: echo -n "found=$(find . -name "*Dockerfile" -printf "${{ inputs.working-directory }}%p ")" >> $GITHUB_OUTPUT
```

Comment:
Another way to produce this is to use the `working-directory` instead of `.` as lookup directory to find
`find ${{ inputs.working-directory }} -name "*Dockerfule" -printf "%p "`

Reply:
The command already runs in the correct working directory, what is required is to include the working directory in the path so that the action that checks the Dockerfile knows where to look. Does that make sense?

Reply:
Resolving for now, please let me know if there is any more feedback!